dt-contacts/contacts-post-type.php

```diff
@@ -491,90 +564,127 @@ public function get_contact_field_defaults( $post_id = null, $include_current_po
             'section'     => 'misc',
             'hidden'      => true
         ];
         $fields["unfollow"] = [
             'name'        => __( 'Un-Follow', 'disciple_tools' ),
-            'description' => 'Users not following this contact',
+            'description' => __( 'Users not following this contact', 'disciple_tools' ),
             'type'        => 'multi_select',
             'default'     => [],
             'section'     => 'misc',
             'hidden'      => true
         ];
-        $fields["tags"] = [
-            "name" => __( "Tags", "disciple_tools" ),
-            "type" => "multi_select",
-            "default" => []
+        $fields['tags'] = [
+            'name'        => __( 'Tags', 'disciple_tools' ),
+            'description' => __( 'A useful way to group related items and can help group contacts associated with noteworthy characteristics. e.g. business owner, sports lover. The contacts can also be filtered using these tags.', 'disciple_tools' ),
+            'type'        => 'multi_select',
+            'default'     => [],
+            'section'     => 'misc',
         ];
+
+        $fields['contact_connections'] = [
+            'name'        => __( 'Contact Connections', 'disciple_tools' ),
+            'description' => 'Connections this contact has with other users (and groups) in this system.',
+            'type'        => 'key_select',
+            'default'     => [
+                'groups' => [
+                    'label' => __( 'Groups', 'disciple_tools' ),
+                    'description' => __( 'The groups that this user is in. Search for a group or create a new group this user will be added to.', 'disciple_tools' ),
+                ],
+                'relationship'   => [
+                    'label' => __( 'Connection or Relation', 'disciple_tools' ),
+                    'description' => __( 'This user is connected or related to these other users.', 'disciple_tools' ),
+                ],
+                'baptized_by'   => [
+                    'label' => __( 'Baptized By', 'disciple_tools' ),
+                    'description' => __( 'Who this user was baptised by.', 'disciple_tools' ),
+                ],
+                'baptized'   => [
+                    'label' => __( 'Baptized', 'disciple_tools' ),
+                    'description' => __( 'People who this user has baptised themseleves.', 'disciple_tools' ),
+                ],
+                'coached_by'   => [
+                    'label' => __( 'Coached', 'disciple_tools' ),
+                    'description' => __( 'The people this user is being coached by.', 'disciple_tools' ),
+                ],
+                'coaching'   => [
+                    'label' => __( 'Coaching', 'disciple_tools' ),
+                    'description' => __( 'Who are the people this user is themseleves coaching.', 'disciple_tools' ),
+                ],
+              ],
+            'section'     => 'info',
+        ];
+
```

Comment:
@mikeallbutt please don't add extra fields. This will create confusion in other places.